libs/domain/picking/src/components/picking/picking.component.spec.ts

```diff
@@ -99,8 +108,10 @@ describe('PickingComponent', () => {
 
     it('should render success message', () => {
       expect(
-        element.renderRoot.querySelector('.picking-complete p')?.textContent
-      ).toBe(i18n('picking.all-items-are-processed!'));
+        element.renderRoot
+          .querySelector('.submit-wrapper button')
+          ?.textContent?.trim()
+      ).toBe(i18n('picking.finish-picking'));
     });
```

Comment:
test description says that you want to see the success message, but the test code is checking just if the button exists. Please, rename the test and check the message and button existing when all items are already picked